IntegrationTests/PrivacyDashboard/PrivacyDashboardIntegrationTests.swift

```diff
@@ -61,7 +61,7 @@ class PrivacyDashboardIntegrationTests: XCTestCase {
             .switchToLatest()
             .filter { $0.trackersBlocked.count > 0 }
             .map { $0.trackers.count }
-            .timeout(10)
+            .timeout(30)
```

Comment:
Not expecting to keep this change, rather just using it so that I have something to open a PR with.